financial-connections/src/main/java/com/stripe/android/financialconnections/features/success/SuccessContent.kt

```diff
@@ -301,14 +304,16 @@ private fun SuccessBody(
         horizontalAlignment = Alignment.CenterHorizontally,
         modifier = modifier,
     ) {
-        Text(
-            text = stringResource(R.string.stripe_success_pane_title),
-            style = typography.headingXLarge,
-            textAlign = TextAlign.Center
+        AnnotatedText(
+            text = customHeading ?: TextResource.StringId(R.string.stripe_success_pane_title),
```

Comment:
Should we do this in the ViewModel and expose them as non-null properties on the view state?

Reply:
good idea! - https://github.com/stripe/stripe-android/pull/9037/commits/7d2d5bfe5ac3705b0b3311ab25b253d0262ada18